pkg/execution/executor/executor.go

```diff
@@ -1908,18 +1963,18 @@ func (e *executor) handleGeneratorInvokeFunction(ctx context.Context, gen state.
 
 	opcode := gen.Op.String()
 	err = e.sm.SavePause(ctx, state.Pause{
-		ID:          pauseID,
-		WorkspaceID: item.WorkspaceID,
-		Identifier:  item.Identifier,
-		GroupID:     item.GroupID,
-		Outgoing:    gen.ID,
-		Incoming:    edge.Edge.Incoming,
-		StepName:    gen.UserDefinedName(),
-		Opcode:      &opcode,
-		Expires:     state.Time(expires),
-		Event:       &eventName,
-		Expression:  &strExpr,
-		DataKey:     gen.ID,
+		ID:                  pauseID,
+		WorkspaceID:         item.WorkspaceID,
+		Identifier:          item.Identifier,
+		GroupID:             item.GroupID,
+		Outgoing:            gen.ID,
+		Incoming:            edge.Edge.Incoming,
+		StepName:            gen.UserDefinedName(),
+		Opcode:              &opcode,
+		Expires:             state.Time(expires),
+		Event:               &eventName,
+		DataKey:             gen.ID,
```

Comment:
Can we keep the expression around?  We're still saving pauses, and without this backcompat is broken.  This means that on rollbacks / rollouts with blue-green deploys we'll match wrong pauses.

Reply:
Sure, we can add that back.